src/odemis/dataio/tiff.py

```diff
@@ -1209,8 +1209,11 @@ def _findImageGroups(das):
     for da in das:
         # check if it can be part of the current group (compare just to the previous DA)
         dims = da.metadata.get(model.MD_DIMS, "CTZYX"[-da.ndim::])
-        if (prev_da is None
```

Comment:
I feel that this code is extra difficult to read because of the inverted logic. Maybe you could inverted back, with something like:
```
if prev_da is not None and prev_da.shape == da.shape...:
    log("merging")
else:
    new group
```

Reply:
I've inverted it, and update comments. should be easier to read